src/test/e2e/api/admin/archive.test.ts

```diff
@@ -142,6 +147,57 @@ test('Should be able to revive toggle', async () => {
         .expect(200);
 });
 
+test('Should disable all environments when reviving a toggle', async () => {
+    await db.stores.featureToggleStore.deleteAll();
+    await db.stores.featureToggleStore.create('default', {
+        name: 'feat-proj-1',
+        archived: true,
+    });
+
+    await db.stores.environmentStore.create({
+        name: 'development',
+        enabled: true,
+        type: 'development',
+        sortOrder: 1,
+    });
+
+    await db.stores.environmentStore.create({
+        name: 'production',
+        enabled: true,
+        type: 'production',
+        sortOrder: 2,
+    });
+
+    await db.stores.featureEnvironmentStore.addEnvironmentToFeature(
+        'feat-proj-1',
+        'default',
+        true,
+    );
+    await db.stores.featureEnvironmentStore.addEnvironmentToFeature(
+        'feat-proj-1',
+        'production',
+        true,
+    );
+    await db.stores.featureEnvironmentStore.addEnvironmentToFeature(
+        'feat-proj-1',
+        'development',
+        true,
+    );
+    await app.request
+        .post('/api/admin/archive/revive/feat-proj-1')
+        .send({})
+        .expect(200);
+
+    await app.request
+        .get(
+            '/api/admin/projects/default/features/feat-proj-1?variantEnvironments=true',
+        )
+        .expect((res) => {
+            const feature = res.body;
+            return feature.environments.every((env) => !env.enabled);
```

Comment:
same here, I'd use expect from jest instead of writing expect in supertest. I normally use supertest expect for HTTP stuff and regular expect for regular assertions since they provide better reporting capabilities.

Reply:
makes sense